src/track/beatutils.h

```diff
@@ -47,6 +47,11 @@ class BeatUtils {
      */
     static double calculateBpm(const QVector<double>& beats, int SampleRate,
                                int min_bpm, int max_bpm);
+
+    // Remove jitter and false beats noise from the beats that make a beatmap
+    static QVector<double> correctBeatmap(
+            QVector<double>& rawBeats, int SampleRate, double minBpm, double maxBpm, bool removeArrythmic);
```

Comment:
```suggestion
            QVector<double>& rawBeats, const SampleRate& sampleRate, double minBpm, double maxBpm, bool removeArrythmic);
```

using enum class/QFlags instead of a bool would also increase readability from the caller's perspective.

Reply:
Do you mean the bool removeArrythmic?
I don't understand how a enum would be helpful, can you clarify?